### DISCUSSION TITLE
Raspberrymatic versehentlich auf LAN-Gateway gesetzt

### DISCUSSION THREAD

**Discussion by UPaulick** [OpenCCU/OpenCCU#1938] [🇩🇪 Fragen&Antworten]
Hallo Zusammen,
ich habe seit ein paar Jahren, zunächst mit CCU2 und dann mit einer RaspberryPI eine CCU3, also eine RapberryMatic am laufen. Bin kein Freak (im positiven Sinn, weil keine Zeit...) und wollte die alte CCU2 als LAN-Gatway einrichten, was nur "zur Hälfte funktionierte". Aus diesem Grunde habe nach Abhilfe gesucht und "in geistiger Umnachtung" mit dem Scipt:

string stdout;
string stderr;
system.Exec('touch /usr/local/HMLGW && reboot', &stdout, &stderr);

über "Programme und Verknüpfungen" meine "eigentliche Hauptstation" in ein LAN-Gateway verwandelt. Nun suche ich nach einer Möglichkeit, das wieder rückgängig zu machen. Habe - SSH ist dort aktiviert - nun mittels Putty versucht, die Datei "/usr/local/HMLGW" zu entfernen. Putty macht mir "die Tür zu meiner CCU auf", aber ich komme aber nicht rein. Das Passwort wird nicht angenommen.
Was kann ich tun, kann mir hier jemand helfen???

Besten Dank in "die Runde", VG U. Paulick

**Comment by Weseler485** [comment]
Hast du das Kennwort genutzt welches du beim Aktivieren von SSH gesetzt hast? Das GUI PW funktioniert da nicht.

**Reply by UPaulick** [reply]
Hallo Maik,

besten Dank! Das Ganze hat sich dank Jens und zwei weiterer hilfreicher Hände gelöst. Habe das dann mit einem direkten Anschluss eines Monitors (dabei gelernt, dass nicht alle Monitore mitspielen - frag mich nicht warum) gelöst. Die Eingabe des SSH-Passwortes funktionierte dann doch - lag wohl an meinen "krummen Fingern" - man sieht ja nicht was man eingibt... (im Falle des Passwortes), keine "Sternchen" o.ä. - so hat man ja nicht einmal die Sicherheit die Anzahl der Zeichen zu checken... da fällt mir ein: Kann man das nicht grundsätzlich ändern? - Oder ist das ein "Dumme Frage"... 

Jedenfalls noch mals besten Dank an Alle!

Gruß Uli
